tcms/testcases/forms.py

```diff
@@ -625,5 +625,3 @@ def populate(self, case_ids=None):
             # note: backwards relationship filter. TestCaseTag -> TestTag
             self.fields['o_tag'].queryset = TestTag.objects.filter(
                 cases__in=case_ids).order_by('name').distinct()
-        else:
-            self.fields['o_category'].queryset = TestCaseCategory.objects.all()
```

Comment:
I get it that CaseTagForm is only used in tcms/testcases/views.py:def tag but you really have to provide an explanation in the commit log when deleting things. That deletion is not related to the replacement of `request.REQUEST` and should have gone into a separate commit.